wagtail_review/migrations/0001_initial.py

```diff
@@ -4,7 +4,24 @@
 
 from django.conf import settings
 from django.db import migrations, models
+from django.db.migrations.recorder import MigrationRecorder
 import django.db.models.deletion
+from wagtail import VERSION as WAGTAIL_VERSION
+
+
+def get_revision_model():
+    revision_model = "wagtailcore.PageRevision"
+    if WAGTAIL_VERSION >= (4, 0, 0):
```

Comment:
I missed one edge case here.
See https://github.com/wagtail/wagtail-localize/pull/643 for the updated version

Reply:
Thanks @zerolab! Have now ported this fix in 8763abe24d7fe1589d279031a4c517f01be4d104.